<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InOutPanel</class>
 <widget class="QGroupBox" name="InOutPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Input / Output</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="inputLayers"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="outputMode"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="outputMessages"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="previewMode"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
